source/crud/tests/v2/bulkWrite-arrayFilters.yml

```diff
@@ -65,3 +65,60 @@ tests:
         data:
           - {_id: 1, y: [{b: 2}, {b: 2}]}
           - {_id: 2, y: [{b: 0}, {b: 2}]}
+  -
+    description: "BulkWrite with arrayFilters using update pipelines"
+    operations:
+      -
+        name: "bulkWrite"
+        arguments:
+          requests:
+            -
+              # UpdateOne when one document matches arrayFilters
+              name: "updateOne"
+              arguments:
+                filter: {}
+                update: [ { $set: { "y.$[i].b": 2 } } ]
```

Comment:
I think it'd make sense for these example to at least specify two updates in the array, instead of wrapping a single update document in a BSON array. If the server has already implemented this feature, there may be an example to use in their code review's JS tests.

Reply:
The new update test file has contrived tests that have two update documents in the array. James Whalin commented that "Given we restrict users to the $addFields, $project and $replaceRoot stages currently, I am not sure if there are any good real-life multi-stage use cases."